src/System.Runtime/samples/DateTime/DateTime.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+
+public class Program
+{
+	public static void Main()
+	{
+		// Demonstrates common uses of DateTime struct.
+		// Doc: https://msdn.microsoft.com/library/system.datetime.aspx
+		
+		// Get and manipulate time
+		Console.WriteLine("Print the DateTime now");
+		Console.WriteLine(DateTime.Now);
+		Console.WriteLine();
+		
+		Console.WriteLine("Print the UTC DateTime now");
+		Console.WriteLine(DateTime.UtcNow);
+		Console.WriteLine();
+		
+		Console.WriteLine("Print the DateTime in 10 mins");
+		Console.WriteLine(DateTime.Now.AddMinutes(10));
+		Console.WriteLine();
+		
+		Console.WriteLine("Print the DateTime 5 days and 4 hours ago");
+		Console.WriteLine(DateTime.Now.AddDays(-5).AddHours(-4));
+		Console.WriteLine();
+		
+		// Create a DateTime for a specific date/time
+		// Uses Unix epoch as the example date - https://en.wikipedia.org/wiki/Unix_time
+		DateTime epoch = new DateTime(1970,1,1);
```

Comment:
We should teach people to never create DateTimeKind.Unspecified values. Would it make sense to use one of the ctors with the DateTimeKind parameter?